### PR TITLE
style(ui): refactor page level styling for both mobile & desktop

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

 refactor page level styling for both mobile & desktop;

image
image
image
image
image
image
image

## Linear Issue Reference

log-2373

## Testing

test locally
@logto-io/eng

### CODE REVIEW

**Comment by demonzoo** [comment]
You can replace the Logto logo, for the current one doesn't work well in dark theme.

**Comment by simeng-li** [comment]
> You can replace the Logto logo, for the current one doesn't work well in dark theme.

It comes from the signin-exp settings. Need to support light/dark mode icons.

**Comment by demonzoo** [comment]
> > You can replace the Logto logo, for the current one doesn't work well in dark theme.
> 
> It comes from the signin-exp settings. Need to support light/dark mode icons.

There's already a dark theme compatible logo in Admin Console. Please check it out

File: packages/ui/src/components/NavBar/index.module.scss [hunk 1]
```diff
@@ -8,12 +8,20 @@
   padding: _.unit(3) _.unit(10);
   position: relative;
 
-  svg {
+  .backBtn {
```
Comment (demonzoo) [inline packages/ui/src/components/NavBar/index.module.scss:11]: `.backButton`

File: packages/ui/src/pages/Consent/index.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-import BrandingHeader from '@/components/BrandingHeader';
 import useApi from '@/hooks/use-api';
 import { PageContext } from '@/hooks/use-page-context';
 
 import * as styles from './index.module.scss';
 
 const Consent = () => {
   const { experienceSettings } = useContext(PageContext);
-  const { logoUrl = '' } = experienceSettings?.branding ?? {};
+  const logoUrl = conditionalString(experienceSettings?.branding.logoUrl);
```
Comment (demonzoo) [inline packages/ui/src/pages/Consent/index.tsx:13]: `conditionalString` is unnecessary

File: packages/ui/src/pages/ErrorPage/index.tsx [hunk 3]
```diff
@@ -21,19 +21,18 @@ const ErrorPage = ({ title = 'description.not_found', message }: Props) => {
...
-          {t('action.back')}
-        </Button>
       </div>
+      <Button
+        className={styles.backBtn}
+        onClick={() => {
+          navigate(-1);
+        }}
+      >
+        {t('action.back')}
```
Comment (demonzoo) [inline packages/ui/src/pages/ErrorPage/index.tsx:34]: This back button looks different from the one on navbar

**Review by demonzoo — APPROVED** [review]
Overall looks good

File: packages/ui/src/components/NavBar/index.module.scss (hunk 1 above)
Comment (simeng-li) [reply packages/ui/src/components/NavBar/index.module.scss:11]: updated

File: packages/ui/src/pages/ErrorPage/index.tsx [hunk 4]
```diff
@@ -21,19 +21,18 @@ const ErrorPage = ({ title = 'description.not_found', message }: Props) => {
...
-          className={styles.backBtn}
-          onClick={() => {
-            navigate(-1);
-          }}
-        >
-          {t('action.back')}
-        </Button>
       </div>
+      <Button
+        className={styles.backBtn}
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply packages/ui/src/pages/ErrorPage/index.tsx:29]: updated

File: packages/ui/src/pages/ErrorPage/index.tsx (hunk 3 above)
Comment (simeng-li) [reply packages/ui/src/pages/ErrorPage/index.tsx:34]: what do you mean by different? One is the page button one is the icon link.  Do you mean text content?

File: packages/ui/src/pages/SocialRegister/index.tsx [hunk 5]
```diff
@@ -21,8 +22,9 @@ const SocialRegister = () => {
 
   return (
     <div className={styles.wrapper}>
-      <NavBar title={t('description.bind_account_title')} />
+      <NavBar title={isMobile ? t('description.bind_account_title') : undefined} />
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply packages/ui/src/pages/SocialRegister/index.tsx:25]: false is not assignable to the title as isMobile is a bool type

File: packages/ui/src/pages/ErrorPage/index.tsx (hunk 3 above)
Comment (demonzoo) [reply packages/ui/src/pages/ErrorPage/index.tsx:34]: Yes, the back button on navbar has icon and text, and text is only available for desktop.
But the back button here only has text. This is the difference I'm talking about

File: packages/ui/src/pages/SocialRegister/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (demonzoo) [reply packages/ui/src/pages/SocialRegister/index.tsx:25]: ok, np

File: packages/ui/src/pages/ErrorPage/index.tsx (hunk 3 above)
Comment (simeng-li) [reply packages/ui/src/pages/ErrorPage/index.tsx:34]: The nav back button is included in the Nav link.  Error page includes both nav bar and a standard back button.

**Pull request merged**